lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb

```diff
@@ -1,661 +1,710 @@
 # -*- coding: binary -*-
+
 require 'rex/post/meterpreter'
 
 module Rex
-module Post
```

Comment:
@cn-kali-team Thanks for the PR; We try to avoid running rubocop on core library files - as it makes viewing Git Blame more time consuming

It would be great to revert running rubocop on this file :+1: